openmc/tallies.py

```diff
@@ -8,6 +8,7 @@
 import sys
 
 import numpy as np
+import h5py
```

Comment:
Ah yes, it looks like the PEP8 Nazid have assimilated me within their
ranks. I'll adjust this to import it as needed.
On Oct 23, 2015 1:35 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/tallies.py
> <https://github.com/mit-crpg/openmc/pull/482#discussion_r42892007>:
>
> > @@ -8,6 +8,7 @@
> >  import sys
> >
> >  import numpy as np
> > +import h5py
>
> Did we want to keep import h5py next to its use? I know someone opined at
> some point that having the import at the module-level makes the entire API
> depend on it, so if you don't have h5py you can't even construct inputs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/482/files#r42892007>.
>